kernel/kernel-api/src/main/java/io/delta/kernel/annotation/Nullable.java

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
For my knowledge, why don't we use existing javax annotations instead of defining new ones in Kernel?

Reply:
Don't want to pull in a new dependency!